doc/classes/Engine.xml

```diff
@@ -41,12 +43,14 @@
 			<return type="float">
 			</return>
 			<description>
+				Return the desired frames per second. If the hardware cannot keep up, this setting may not be respected. It defaults to 0, which indicates no limit.
 			</description>
 		</method>
 		<method name="get_time_scale">
 			<return type="float">
 			</return>
 			<description>
+				Controls how fast or slow the in-game clock ticks versus the real life one. It defaults to 1.0. A value of 2.0 means the game moves twice as fast as real life, whilst a value of 0.5 means the game moves at half the regular speed.
```

Comment:
The "get" method does not control the time scale, just returns the current value.